packages/cli/src/commands/generate/templates/cell/test.js.template

```diff
@@ -1,10 +1,31 @@
-import React from 'react';
-import { fireEvent, cleanup } from '@testing-library/react'
+import { render, cleanup } from '@testing-library/react'
 
-import ${pascalName}Cell from './${pascalName}Cell'
+import { Loading, Empty, Failure, Success } from './${camelName}Cell'
 
-describe('${pascalName}Cell', () => {
-  it('is true', () => {
-    expect(true).toBe(true)
+const error = { message: 'error message' }
+
+describe('${camelName}Cell', () => {
+  afterEach(() => {
+    cleanup()
+  })
+  it('Loading renders successfully', () => {
+    expect(() => {
+      render(<Loading />)
+    }).not.toThrow()
+  })
+  it('Empty renders successfully', () => {
+    expect(() => {
+      render(<Empty />)
+    }).not.toThrow()
+  })
+  it('Failure renders successfully', () => {
+    expect(() => {
+      render(<Failure error={error} />)
+    }).not.toThrow()
+  })
+  it('Success renders successfully', () => {
+    expect(() => {
+      Success('${camelName}Cell')
```

Comment:
Success is given an object that it then destructures, looking for a key with the `camelName` version of the cell itself. So this would need to be more like:

    Success({ ${camelName}: { foo: 'bar' } })

And then the render should output `{"foo": "bar"}`

Reply:
Ah, I did not understand that. Fixed!

Reply:
I think you still want to `render` this one though?